### DISCUSSION TITLE
How to display unchecked state for rating input?

### DISCUSSION THREAD

**Discussion by crapthings** [saadeghi/daisyui#1665] [Q&A]
<img width="570" alt="image" src="https://user-images.githubusercontent.com/1147704/223132823-e68f1c82-1f31-44c2-843c-f1f819860d82.png">

this one checked all

```jsx
<div className="rating">
  <input type="radio" name="rating-2" className="mask mask-star-2 bg-primary" />
  <input type="radio" name="rating-2" className="mask mask-star-2 bg-primary" />
  <input type="radio" name="rating-2" className="mask mask-star-2 bg-primary" />
  <input type="radio" name="rating-2" className="mask mask-star-2 bg-primary" />
  <input type="radio" name="rating-2" className="mask mask-star-2 bg-primary" />
</div>
```

**Comment by saadeghi** [comment]
Can you explain more? I'm not sure what you mean...

Please tell me what is your expected behavior and what is the current behavior

**Reply by crapthings** [reply]
like this if no checked

<img width="171" alt="image" src="https://user-images.githubusercontent.com/1147704/223419927-63574bd6-09a2-4dbd-89fd-ae85dff434ea.png">
